topologies/kne/ixia-arista-3ports.textproto

```diff
@@ -0,0 +1,83 @@
+name: "ixia-c"
```

Comment:
Let's drop the 3ports variant and just keep the 4ports one?

Reply:
Agreed